util/util.go

```diff
@@ -732,7 +732,7 @@ func RemountVolume(volumeName string) error {
 	res = strings.TrimSpace(res)
 	if res == "" {
 		logrus.Infof("Cannot find any mount record for volume %v, will skip remounting it", volumeName)
-		return nil
+		return fmt.Errorf("cannot find any mount record for volume %v", volumeName)
```

Comment:
What if a volume containing the EXT4 filesystem is attached manually but not mounted? In this case, can we return an error here?
Similar below.

Reply:
If the volume is attached manually but not mounted before, is it correct that we should not remount it? If it is correct then I think that we should return error from RemountVolume() because we skipped remounting the volume.